### ISSUE TITLE
Allow node engine v13

### ISSUE THREAD

**Issue by jonnybel** [testing-library/pptr-testing-library#31]
On upgrading to version `0.5.0` this module started giving the following error on our project which uses node 13.12
```
error pptr-testing-library@0.5.0: The engine "node" is incompatible with this module. Expected version "^10 || ^12". Got "13.12.0"
error Found incompatible module.
```

Previously, it would work without problems.

I believe this was introduced while removing support for node 8 in #24, which restricted the module for node versions 10 and 12, although version 13 also seems to work without problems, so perhaps it can be added to the `engines` list?

Thank you!

**Comment by patrickhulce** [comment]
Sorry for the super late response here but we've significantly loosened the engines definition now, anything >=10 matches :)

**Comment by jonnybel** [comment]
Thanks, I'll be waiting for a new release then :)

**Comment by patrickhulce** [comment]
It was released awhile ago 0.6.0 should already have it :)

**Comment by jonnybel** [comment]
Sorry, I tried version 0.6.0 and still got the error...

Perhaps I am missing something?

If I understand correctly, the PR with the fix for this Issue was merged in June (https://github.com/testing-library/pptr-testing-library/pull/34)
But version 0.6.0 was released back in May (https://github.com/testing-library/pptr-testing-library/releases/tag/v0.6.0) so it doesn't include the fix.

**Comment by patrickhulce** [comment]
Doh, you're right there were multiple engines changes I merged them in my head sorry!

**Comment by patrickhulce** [comment]
Just published 0.6.1 with #34 👍

**Comment by jonnybel** [comment]
No worries 😄 

Thanks a lot!
